k8s/helmfile/helmfile.yaml

```
@@ -209,7 +209,7 @@ releases:
   - name: wbaas-backup
     namespace: default
     chart: wbstack/wbaas-backup
-    version: 0.0.6
+    version: {{ ternary "0.1.0" "0.0.6" (ne .Environment "production") }}
```

missing `.Name` for `.Environment`
```suggestion
    version: {{ ternary "0.1.0" "0.0.6" (ne .Environment.Name "production") }}
```